sources/packages/api/src/services/student/student.service.ts

```diff
@@ -261,13 +261,13 @@ export class StudentService extends RecordDataModelService<Student> {
     lastName: string,
     appNumber: string,
   ): Promise<Student[]> {
-    const applicationExistsQuery = this.applicationRepo
-      .createQueryBuilder("application")
-      .where("application.applicationNumber Ilike :appNumber")
-      .andWhere("application.applicationStatus != :overwrittenStatus")
-      .select("1");
     const searchQuery = this.repo
       .createQueryBuilder("student")
+      .leftJoinAndSelect(
+        Application,
+        "application",
+        "application.student_id = student.id",
```

Comment:
Please do not use database columns names on the queries.

Reply:
Feedback comment addressed.